src/test/java/picard/util/FifoBufferTest.java

```diff
@@ -57,8 +57,8 @@ public void test(final double megabytes) throws IOException {
         }
 
         // Run the input file through the FifoBuffer and back into another file
-        final Md5CalculatingInputStream in   = new Md5CalculatingInputStream(new FileInputStream(inputFile), null);
-        final Md5CalculatingOutputStream out = new Md5CalculatingOutputStream(new FileOutputStream("/dev/null"), null);
+        final Md5CalculatingInputStream in   = new Md5CalculatingInputStream(new FileInputStream(inputFile), (File)null);
+        final Md5CalculatingOutputStream out = new Md5CalculatingOutputStream(new FileOutputStream("/dev/null"), (File)null);
```

Comment:
I don't want to use a snapshot here, only official releases. 

Strange about the tests not failing here..I agree.